### PR TITLE
Added recipe for lazy

### PR DESCRIPTION
**Pull request by pmlandwehr**
This recipe is super slack with license info. All that I can find is what's provided on PyPI

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Hey @stefanholek, I'm building a copy of `lazy` for conda-forge, but haven't been able to find additional license info for the package. Can you specify *which* BSD license you were using?

File: recipes/lazy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - python
+
+test:
+  imports:
+    - lazy
+    - lazy.tests
+
+about:
+  home: https://github.com/stefanholek/lazy
+  license: BSD
```
Comment (ocefpaf) [inline recipes/lazy/meta.yaml]: Argh!!! https://github.com/stefanholek/lazy/issues/2

Comment (jakirkham) [inline recipes/lazy/meta.yaml]: 😢 

Maybe we should just merge it then. The author seems totally unresponsive. What more can we do?

**Comment by jakirkham** [comment]
Thanks @pmlandwehr . Sorry for the hold-up.

**Pull request merged**